### PR TITLE
Added different level of logs in the rp-ivr-webhook-handler service

### PR DESCRIPTION
**Pull request by Sachinbisht27**
[#401](https://github.com/DostEducation/RP_IVR_analytics/issues/401)

### Types of changes

- [ ] Bug fix (a change which fixes an issue)
- [x] New feature (a change which adds functionality)

### Short description of what this resolves:
**Describe your changes in detail**
Currently, the service is not able to generate different levels of logs in the cloud function. 
I have added different levels of logs in the rp-ivr-webhook-handler service.

### Checklist:

- [x] I have performed a self-review of my own code.
- [x] The code follows the style guidelines of this project.
- [ ] I have documented my code wherever required.
- [ ] The changes requires a change to the documentation.
- [ ] I have updated the documentation based on the my changes.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
-    parent = client.queue_path(PROJECT, LOCATION, QUEUE)
-    task = {
-        "http_request": { # Specify the type of request.
-            "http_method": tasks_v2.HttpMethod.POST,
-            "url": TARGET_FUNCTION_URL, # The full url path that the task will be sent to.
-            "body": payload.encode('utf-8'),
-            "headers": {"Content-type": "application/json"}
+    try:
+        payload = json.dumps(req.get_json())
+        logger.info(f"Payload: {payload}")
```
Comment (Satendra-SR) [inline main.py]: Let's remove this logger, not sure if we really need to log this

File: main.py [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
+
+        client = tasks_v2.CloudTasksClient()
+        parent = client.queue_path(config.PROJECT, config.LOCATION, config.QUEUE)
+        task = {
+            "http_request": { # Specify the type of request.
+                "http_method": tasks_v2.HttpMethod.POST,
+                "url": config.TARGET_FUNCTION_URL, # The full url path that the task will be sent to.
```
Comment (Satendra-SR) [inline main.py]: Let's remove the comment
```suggestion
                "url": config.TARGET_FUNCTION_URL,
```

File: main.py (hunk 1 above)
Comment (Satendra-SR) [reply main.py]: Instead, let's log the payload only when we are getting some exceptions, under **except** block

File: utils/loggingutils.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import logging
+import config
+from google.cloud import logging as gcloud_logging
+
+logger = logging.getLogger()
+logging.basicConfig(level=logging.DEBUG)
```
Comment (Satendra-SR) [inline utils/loggingutils.py]: What do you think, should we use the same logging level or change it for production environment?

File: main.py (hunk 1 above)
Comment (Sachinbisht27) [reply main.py]: Sure, instead of logging the payload every time. We will log it only when any exception occurs.

File: main.py (hunk 2 above)
Comment (Sachinbisht27) [reply main.py]: Sure, removing it.

File: utils/loggingutils.py (hunk 3 above)
Comment (Sachinbisht27) [reply utils/loggingutils.py]: I used the DEBUG because it is the basic level of logging but in production, we can use the INFO logging level.

**Review by Satendra-SR — APPROVED** [review]

**Pull request merged**